frontend/pages/Kolide500/Kolide500.jsx

```diff
@@ -18,15 +78,23 @@ class Kolide404 extends Component {
           <h1>Uh oh!</h1>
           <h2>Error 500</h2>
           <p>Something went wrong on our end.</p>
-          <p>We have alerted the engineers and they are working on a solution.</p>
+          {renderError()}
+          <p>Please file an issue if you believe this is a bug.</p>
+          <a href="https://github.com/kolide/fleet/issues">File an issue</a>
```

Comment:
```suggestion
          <a href="https://github.com/fleetdm/fleet/issues">File an issue</a>
```

Change issue link to new fork :)

Reply:
Ah, great catch. I remember making a mental note to commit this change and totally forgot.

Reply:
PR name and description are updated as well.